src/app/Http/Controllers/API/PresentationController.php

```diff
@@ -96,7 +96,12 @@ public function show(Presentation $presentation)
      */
     public function update(Request $request, Presentation $presentation)
     {
-        //
+        $request->validate([
+            'title' => 'sometimes|required',
+            'date' => 'sometimes|required|date',
+        ]);
```

Comment:
Why are you using `sometimes`? The field are required aren't they?

Reply:
`sometimes` is used to check if the request contains the param, otherwise it will not be modifed
That way you could for send a request to change the date without specifying the title in your request.
Without the `sometimes` validation it's mandatory to fill all params even if they haven't been changed
more details here: https://laravel.com/docs/6.x/validation#conditionally-adding-rules